mmseg/core/__init__.py

```diff
@@ -1,4 +1,7 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 from .evaluation import *  # noqa: F401, F403
+from .layer_decay_optimizer_constructor import \
+    LayerDecayOptimizerConstructor  # noqa: F401, F403
+from .resize_transform import SETR_Resize  # noqa: F401, F403
```

Comment:
`# noqa: F401, F403` is ignore Flake8  F401 and F403. F401 is 'module imported but unused', and F403 is 'unable to detect undefined names '. I think F403 is useless here